09week/checkpoint3/public/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>My Trello App</title>
+
+    <script src="http://code.jquery.com/jquery-1.7.1.min.js"></script>
```

Comment:
what are you using jquery for? you shouldn't have to use it if you're using react